### DISCUSSION TITLE
How do you locate a binary?

### DISCUSSION THREAD

**Discussion by drauschenbach** [goreleaser/nfpm#641] [Q&A] [answered]
I must be missing something obvious, but how do I locate a compiled binary? I don't see any way to reference `${GOPATH}/bin` or `$(go env GOPATH)/bin`.

**Comment by caarlos0** [comment, marked as answer]
either use something like `envsubst` or copy the binary to a known place before running nfpm.

fwiw, since you seem to be doing it with Go binaries, https://goreleaser.com does it automatically.
